### ISSUE TITLE
Virus transports mission and Trader assault mission mixed

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/open-ig#776]
```
Game version:
0.95.145

Operating System:
Windows 7 x86

Java runtime version: (run java -version)
1.7.0_25

Installed using the Launcher?
Yes

Game language :
FR


What steps will reproduce the problem?

1. I managed the first and the second infection.
2. After the second infection, I warned the Colonel.

Just after having located the virus' transports, I got a Help request from a 
Trader attacked by Garthog near Achiles or Centronom I think.

3. I decided to help the trader first.


When I arrived, the Garthog fleet had a strange behaviour. They just went 
through the screen and fled. Also, the only message I could send was like "Die 
Virus Transport"

I think that the game was confused and mixed the two mission.


I am too far in the game now to try to reproduce the event shortly, but you can 
find in attachment a reproduction on what it looked like in the spacebattle 
screen with the "orders" option available.

```

Original issue reported on code.google.com by `benjamin...@gmail.com` on 1 Sep 2013 at 3:28

Attachments:
- [confusing battle against garthorg.png](https://storage.googleapis.com/google-code-attachments/open-ig/issue-776/comment-0/confusing battle against garthorg.png)

**Comment by GoogleCodeExporter** [comment]
```
Seen as well.
```

Original comment by `anonymis...@arcor.de` on 1 Sep 2013 at 4:24

Attachments:
- [save-2013-09-01-18-20-31-421.xml.gz](https://storage.googleapis.com/google-code-attachments/open-ig/issue-776/comment-1/save-2013-09-01-18-20-31-421.xml.gz)

**Comment by GoogleCodeExporter** [comment]
```
They are unarmed transports, so they naturally just want to get through. It is 
the easiest battle of the game. Just do as the objective says and destroy them. 
If you let them pass, they will eventually reach New Caroline and game over.
```

Original comment by `akarn...@gmail.com` on 1 Sep 2013 at 4:48
- Changed state: **Closed**

**Comment by GoogleCodeExporter** [comment]
```
Please don't go over this too fast. The problem here is that the AI applies the 
virus carriers combat tactics to a normal fleet, did you get that ?
```

Original comment by `anonymis...@arcor.de` on 2 Sep 2013 at 6:53

**Comment by GoogleCodeExporter** [comment]
```
Sorry, the I didn't understand the issue. So the issue is that while you had 
the virus carriers in sight, another Garthog attack happened and they just 
moved across the screen.
```

Original comment by `akarn...@gmail.com` on 2 Sep 2013 at 8:07
- Changed state: **Accepted**
- Added labels: **Missions**

**Comment by GoogleCodeExporter** [comment]
```
Should be fixed in version 0.95.148.
```

Original comment by `akarn...@gmail.com` on 3 Sep 2013 at 8:56
- Changed state: **Fixed**
